### PR TITLE
Pre mainnet genesis

### PR DESCRIPTION
**Pull request by jolestar**
## Summary

[genesis] Set main genesis bitcoin block and release main genesis

### CODE REVIEW

File: crates/rooch-types/src/genesis_config.rs [hunk 1]
```diff
@@ -150,24 +150,32 @@ pub static G_TEST_CONFIG: Lazy<GenesisConfig> = Lazy::new(|| {
     }
 });
 
+//curl -sSL https://mempool.space/api/block/$(curl -sSL https://mempool.space/api/block-height/852203)/header
+
+static MAIN_GENESIS_HEIGHT_HEADER: Lazy<(u64, Header)> = Lazy::new(|| {
+    (852203, bitcoin::consensus::deserialize(
```
Comment (popcnt1) [inline crates/rooch-types/src/genesis_config.rs:156]: Shouldn't this be 852202?

Comment (jolestar) [reply crates/rooch-types/src/genesis_config.rs:156]: The Bitcoin block will execute from 852203, include 852203

File: crates/rooch-types/src/genesis_config.rs [hunk 2]
```diff
@@ -150,28 +150,33 @@ pub static G_TEST_CONFIG: Lazy<GenesisConfig> = Lazy::new(|| {
...
+        &hex::decode("0000002aeab3d78df1e6bdd612df31107b32470b2946758410920100000000000000000012c85f974f244dce28142748dd6dc0a8d2a8cdca6706442e6df4b3b52c0d985d794c94666d8a031729521ec6")
+            .expect("Should be valid"),
+    ).expect("Should be valid"))
+});
+
+pub static G_MAIN_CONFIG: Lazy<GenesisConfig> = Lazy::new(|| GenesisConfig {
```
Comment (baichuan3) [inline crates/rooch-types/src/genesis_config.rs:162]: geneis_config 是不是应该放到 rooch-config里？

**Review by popcnt1 — APPROVED** [review]

File: crates/rooch-types/src/genesis_config.rs (hunk 2 above)
Comment (jolestar) [reply crates/rooch-types/src/genesis_config.rs:162]: GenesisConfig 和 ChainID 以及 RoochNetwork 是关联的。

**Pull request merged**